internal/lib/lib.go

```diff
@@ -1397,6 +1407,14 @@ func GetAKOIDPrefix() string {
 	}
 	return akoID
 }
+
+//TODO: Optimize
+func IsNamespaceBlackListed(namespace string) bool {
+	nsBlockedList := AKOControlConfig().GetAKOBlackListedNamespaces()
```

Comment:
I think it will be better to avoid calling GetAKOBlacklistedNamespaces() every time instead call the map created out from `BLOCKED_NS_LIST` that way we can avoid unmarshaling the list every time.

Reply:
I did not get your comment about un-marshalling. My intention was not to expose variable outside its required scope so introduced this function and reuse value in other code part if require. As part of optimization, I can introduce local variable here and check if it is set or not.

Reply:
sure, we can take those changes in the next release as an optimization.